### DISCUSSION TITLE
v2.16.4-beta

### DISCUSSION THREAD

**Discussion by Mygod** [Mygod/VPNHotspot#485] [General]
Fixes.

<hr /><em>This discussion was created from the release <a href='https://github.com/Mygod/VPNHotspot/releases/tag/v2.16.4'>v2.16.4-beta</a>.</em>

**Comment by zzduci** [comment]
![image](https://github.com/Mygod/VPNHotspot/assets/122016328/7e1a5764-56e4-43b2-8192-85c12ded306d)
Can I set this option to automatically open when opening the app? Every time I open the app, I need to manually open the option, otherwise VPN sharing will not take effect
